domains/nithin.json

```diff
@@ -0,0 +1,11 @@
+{
+        "owner": {
+           "username": "nithinlycanz",
+           "email": "nithinnikhil582@gmail.com",
+           "discord": "514353302675259412"
+        },
+
+        "record": {
+            "CNAME": "nithinlycanz.github.io"
```

Comment:
Redirect backs to a is-a.dev domain. No content displayed. Invalid.

Reply:
please explain what i did wrong here